CDP4Grapher/ViewModels/GrapherRibbonViewModel.cs

```diff
@@ -34,11 +36,18 @@ namespace CDP4Grapher.ViewModels
 
     using CDP4Dal;
 
+    using NLog;
+
     /// <summary>
```

Comment:
I did commented on some missing [ms] strings. But I believe it's the other way around. When using the "g" format specifier on a TimeSpan specifying the unit doesn't make sense anymore.